src/docs/get-started/install/_ios-setup.md

```diff
@@ -86,16 +86,21 @@ follow these steps:
 ### Deploy to iOS devices
 
 To deploy your Flutter app to a physical iOS device
-you need the third-party CocoaPods dependency manager
-and an Apple Developer account. You'll also need
-to set up physical device deployment in Xcode.
+you'll need to set up physical device deployment in Xcode
+and an Apple Developer account. If your app is using Flutter plugins,
+you will also need the third-party CocoaPods dependency manager.
 
- 1. Install and set up CocoaPods by running the following commands:
+ 1. You may skip this step if your apps do not depend on
+ [Flutter plugins][] with native iOS code.
+ [Install and set up CocoaPods][] by running the following commands:
 
     ```terminal
     $ sudo gem install cocoapods
-    $ pod setup
     ```
+{{site.alert.note}}
```

Comment:
@sfshaza2 Would you mind deploying this so we can check if this note messed up the numbering?  I know we've needed to add `ol`s in a few places.

Reply:
random tangent: you don't need to deploy it to check formatting. You can just follow https://github.com/flutter/website#developing locally.

Reply:
@xster is correct! Just run the script in tool/serve.sh and you can view it on localhost. You just can't share that link with others.  :)

Reply:
I'm not willing to install nvm and rvm on my development machine.  See: this very diff for all the versioning trouble it causes.

Reply:
ya, it's definitely a hassle. Though on the flip side, fwiw, I never managed to get ruby or python with a reasonably high version to install nicely on mac either without their own ruby or python specific version manager.

Reply:
You're right, I didn't need `rvm`, I was just missing `jekyll` and `npx`.

Reply:
This had two #1s, I changed it to `ol` and `li`s, now it increments to 1 and 2, and the alphabetized `{: type="a"}` also looks right.
<img width="265" alt="Screen Shot 2020-09-25 at 6 40 08 PM" src="https://user-images.githubusercontent.com/682784/94327266-9761d000-ff5e-11ea-92b8-fc42317972f9.png">